rocAL/rocAL/source/api/rocal_api_data_loaders.cpp

```diff
@@ -106,6 +106,40 @@ auto convert_decoder_mode= [](RocalDecodeDevice decode_mode)
     }
 };
 
+auto convert_color_format_to_layout = [](RocalImageColor color_format, size_t n, size_t h, size_t w, size_t c) {
+    switch(color_format) {
+        case ROCAL_COLOR_RGB24:
+        case ROCAL_COLOR_BGR24: {
+            std::vector<size_t> dims_vector = {n, h, w, c};
+            return std::make_tuple(RocalTensorlayout::NHWC, dims_vector);
+        }
+        case ROCAL_COLOR_U8:
+        case ROCAL_COLOR_RGB_PLANAR: {
+            std::vector<size_t> dims_vector = {n, c, h, w};
+            return std::make_tuple(RocalTensorlayout::NCHW, dims_vector);
+        }
+        default:
+            THROW("Unsupported Image type" + TOSTR(color_format))
+    } 
+};
+
+auto convert_color_format_to_sequence_layout = [](RocalImageColor color_format, size_t n, size_t h, size_t w, size_t c, size_t f) {
+    switch(color_format) {
+        case ROCAL_COLOR_RGB24:
+        case ROCAL_COLOR_BGR24: {
+            std::vector<size_t> dims_vector = {n, f, h, w, c};
```

Comment:
rename to dims

Reply:
This file is not a part of this PR. Will be changed in the upcoming PR for readers and meta nodes.